tests/dump_alg_info.c

```diff
@@ -54,6 +54,25 @@ int main(void) {
 		printf("    length-signature: %zu\n", sig->length_signature);
 		OQS_SIG_free(sig);
 	}
+
+	// iterate through stateful signature schemes and print info
+	printf("SIG_STFLs:\n");
+	for (size_t i = 0; i < OQS_SIG_STFL_algs_length; i++) {
+		const char *sig_name = OQS_SIG_STFL_alg_identifier(i);
+		printf("  %s:\n", sig_name);
+		OQS_SIG_STFL *sig = OQS_SIG_STFL_new(sig_name);
+		if (sig == NULL) {
+			printf("    isnull: true\n");
```

Comment:
What is the meaning of this output? Something like "is enabled?" If so, wouldn't such output be more informative?

Reply:
This output is in line with what's currently in place for KEMs and signature schemes, so I think that addressing this is out of scope.